services/src/main/java/org/jboss/windup/web/services/rest/graph/AbstractGraphResource.java

```diff
@@ -96,12 +104,56 @@ private void addEdges(Map<String, Object> result, Vertex vertex, Integer remaini
         return result;
     }
 
-    protected GraphContext getGraph(Long executionID)
+    /**
+     * Opens a graph for given execution.
+     */
+    protected GraphContext getGraphContext(Long executionID)
     {
 
-        WindupExecution execution = entityManager.find(WindupExecution.class, executionID);
-        Path graphPath = Paths.get(execution.getGroup().getOutputPath()).resolve(GraphContextFactory.DEFAULT_GRAPH_SUBDIRECTORY);
+        //WindupExecution execution = entityManager.find(WindupExecution.class, executionID);
+        //Path graphPath = Paths.get(execution.getGroup().getOutputPath()).resolve(GraphContextFactory.DEFAULT_GRAPH_SUBDIRECTORY);
+        //return graphCache.getGraph(graphPath);
+
+        WindupExecution execution;
+        if (executionID == null || executionID == 0)
+            execution = getAnyExecution();
```

Comment:
Yes, a test code. I use this because the first execution ID wasn't always 1.. and I just needed any data. Will go away soon.